class.jetpack.php

```diff
@@ -7474,3 +7513,5 @@ public static function show_backups_ui() {
 		return Jetpack::is_plugin_active( 'vaultpress/vaultpress.php' ) || apply_filters( 'jetpack_show_backups', true );
 	}
 }
+
+Jetpack::get_assumed_site_creation_date();
```

Comment:
Is this just a bit of leftover code?

Reply:
Yeah, cleaned up in https://github.com/Automattic/jetpack/pull/11450/commits/8af97d252b0f3dd0db2c60120c0fcbf6af54a82d.